.travis.yml

```diff
@@ -5,6 +5,8 @@ services:
 
 language: node_js
 
+warnings_are_errors: true
```

Comment:
It'd be great to have a PR with just this.

Reply:
I undo commit #https://github.com/code-for-canada/project-thundercat/pull/204/commits/2c212fd5efce0653c263d5760a0bcb103d91fe07, so this current PR will be used to add this config to Travis, as suggested.